### PR TITLE
DOC: adding example to decimate function

### PR DESCRIPTION
**Pull request by pantelisantonoudiou**

### CODE REVIEW

File: scipy/signal/signaltools.py [hunk 1]
```diff
@@ -4395,6 +4395,43 @@ def decimate(x, q, n=None, ftype='iir', axis=-1, zero_phase=True):
...
+    >>> y = np.cos(x*np.pi*freq*2)
+    
+    Decimate cosine wave.
+    
+    >>> ydem = signal.decimate(y, q)
+    >>> xnew = np.linspace(0, wave_duration, samples_decimated, endpoint=False)
+    
+    Plot original and decimated waves.
+    
+    >>> import matplotlib.pyplot as plt
```
Comment (rgommers) [inline scipy/signal/signaltools.py:4428]: small style issue here: can you put this import at the top of the example, together with the other import?

**Review by rgommers — COMMENTED** [review]
Thanks @pantelisantonoudiou! The example looks good to me from a quick browse. The linting CI job is not too happy with whitespace at the end of lines, there's a bunch of complaints like:
```
scipy/signal/signaltools.py:4398:1: W293 blank line contains whitespace
```
Could you fix those?

File: scipy/signal/signaltools.py (hunk 1 above)
Comment (pantelisantonoudiou) [reply scipy/signal/signaltools.py]: Yes, definitely.  I have removed whitespaces and grouped the imports together.

**Comment by larsoner** [comment]
Azure failure is unrelated. For some reason CircleCI didn't build your latest commit, but the previous one [looked good](https://37893-1460385-gh.circle-artifacts.com/0/html-scipyorg/reference/generated/scipy.signal.decimate.html#scipy.signal.decimate). Looks like you addressed @rgommers 's comments so I'll go ahead and merge. Thanks @pantelisantonoudiou !

**Pull request merged**